### ISSUE TITLE
[Feedback]: Account Type Capability Ambiguous

### ISSUE THREAD

**Issue by parseb** [matter-labs/zksync-docs#148]
### Page

https://docs.zksync.io/build/developer-reference/account-abstraction

### Description

The text mentions EOAs, Contract accounts and then jumps to smart accounts. It does not seem clear for me if I can sponsor transactions for EOAs as most projects rely on existing ownership relations (msg.sender).

The major question being, can I sponsor a transaction for an EOA while keeping msg.sender the EOA?
Feedback might be rushed, did not ponder, but first read, it was not clear for me.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Romsters** [comment]
@parseb yes, you can use [paymaster](https://docs.zksync.io/build/developer-reference/account-abstraction/paymasters). Paymaster can sponsor any transaction, transaction sender remains unchanged.
